tests/end2endtest/es_server.yml

```diff
@@ -0,0 +1,87 @@
+# Easy-server server file for pywbem end2end tests
+#
+# This file has the following format:
+#
+# vault_file: VAULT_FILE           # Path name of easy-server vault file.
+#                                  # Mandatory.
+#
+# default: NICKNAME                # Nickname of default group or server.
+#                                  # Mandatory.
+#
+# server_groups:                   # WBEM server groups.
+#                                  # Optional, default: no groups defined.
+#
+#   NICKNAME42:                    # Nickname of WBEM server group.
+#
+#     - NICKNAME1                  # List of nicknames of servers or server
+#                                  # groups in that group.
+# servers:
+#
+#   NICKNAME1:                     # Nickname of the WBEM server.
+#
+#     description: TEXT            # Short description of the WBEM server.
+#                                  # Mandatory.
+#
+#     contact_name: TEXT           # Name of technical contact for the server.
+#                                  # Optional, default: null.
+#
+#     access_via: TEXT             # Short reminder on the network/firewall/
+#                                  # proxy/vpn used to access the server.
+#                                  # Optional, default: null.
+#
+#     user_defined:
+#
+#       docker_image: ORG/IMAGE:M.N.U
+#                                  # Docker image to pull for setting up the
+#                                  # WBEM server, or null to not pull any
+#                                  # Docker image.
+#                                  # Optional, default: null.
+#
+#       docker_port_mapping:       # Port mapping for Docker image.
```

Comment:
The port mapping confuses me:

1. What is straight mapping? I assume it is same port on both host and docker.
2. How do you map multiple ports (ex 5988 and 5989)

Reply:
1. correct.
2. you cannot. Each server has a single URL, so you need to decide which protocol and port you use. If you want to test both protocols and ports, define two servers.